doc/reference/trd104-syscalls.md

```diff
@@ -496,14 +496,19 @@ failure variant of `Failure`, with an associated error code of
 handle userspace/kernel mismatches should be able to handle `Failure` in
 addition to the expected failure variant (if different than `Failure`).
 
-4.3.1 Command Identifier 0
---------------------------------
 
-Every device driver MUST implement command number 0 as the
-"exists" command.  This command always returns `Success`. This command
-allows userspace to determine if a particular system call driver is
-installed; if it is, the command returns `Success`. If it is not, the
-kernel returns `Failure` with an error code of `NODEVICE`.
+4.3.1 Command Idenfitier 0
+--------------------------
+
+ - [Added August, 2023](../rfcs/2023-08-18--CommandZeroSemantics.md).
```

Comment:
What does this mean to be added August 23? I'm not sure how future readers should interpret this link.

Reply:
I wanted to in some way link the context of this discussion without dumping a whole lot of text into the TRD. I updated that to `[Design RFC](../rfcs/2023-08-18--CommandZeroSemantics.md)`